action.yaml

```
@@ -48,6 +48,6 @@ branding:
 
 runs:
   using: "docker"
-  image: "docker://gcr.io/openssf/scorecard-action:latest"
+  image: "docker://gcr.io/openssf/scorecard-action:v2.0.0-beta1"
```

```suggestion
  image: "docker://gcr.io/openssf/scorecard-action:v2.0.0-beta.1"
```

Needs a `.` after `beta` to be a SemVer-compliant tag.